### ISSUE TITLE
[amp-analytics] inconsistent spec for duration metrics

### ISSUE THREAD

**Issue by lannka** [ampproject/amphtml#6340]
As shown in the spec [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#first-seen-time), we have a bunch of duration metrics for visibility tracking.

However, I found they're calculated relatively to different starting point.

`firstSeenTime` and `firstVisibleTime ` are
> since the trigger is registered by amp-analytics

`lastSeenTime ` and `lastVisibleTime` are
> for the last time since javascript load

`totalTime`
> The value is calculated from the time document became interactive.
 

I'm not sure what's the difference between "trigger registered" vs "javascript load". I saw it's the same piece of code. Maybe we should just consolidate the language.

The bigger question is: why don't we just make all those durations relative to the document load (i.e., become interactive)?

**Comment by dvoytenko** [comment]
This is a sound reasoning. @avimehta can you chime in if we are not looking at this right.

**Comment by rudygalfi** [comment]
@lannka Can you close this and/or reference relevant visibility v2 issues that are addressing this?

**Comment by lannka** [comment]
this is not addressed in visibility v2. we can go ahead and make the change once we all agreed on the same measurement.

**Comment by dvoytenko** [comment]
@lannka Is there anything else left to do here?

**Comment by lannka** [comment]
@dvoytenko yes, nothing has been done yet :-).  we need to first agree on the proposal.
